src/Avalonia.Base/Input/PointerPoint.cs

```diff
@@ -33,55 +33,55 @@ public PointerPoint(IPointer pointer, Point position, PointerPointProperties pro
     /// <summary>
     /// Provides extended properties for a PointerPoint object.
     /// </summary>
-    public struct PointerPointProperties
+    public record PointerPointProperties
```

Comment:
It changes "struct" type to a "class" type. I don't think it's intended and probably shouldn't be part of this PR.